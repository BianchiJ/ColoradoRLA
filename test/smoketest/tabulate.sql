SELECT ccd.description AS contest, 
        ccd.choice AS candidate, 
        count(ccic.cvr_contest_info_id) as votes
    FROM contest_choice_description AS ccd 
        LEFT JOIN ((contest AS c 
            LEFT JOIN cvr_contest_info AS cci
                ON c.id = cci.contest_id)
        LEFT JOIN 
            cvr_contest_info_choice as ccic 
            ON ccic.cvr_contest_info_id = cci.id)
            ON ccd.contest_id = c.id 
        AND ccd.choice = ccic.choice
    GROUP BY contest, candidate
    ORDER BY contest, candidate
;
