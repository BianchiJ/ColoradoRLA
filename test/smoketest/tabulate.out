SELECT ccd.contest_id AS contest, 
        ccd.choice AS candidate, 
        count(ccic.cvr_contest_info_id) as votes
    FROM contest_choice_description AS ccd 
        LEFT JOIN ((contest AS c 
            LEFT JOIN cvr_contest_info AS cci
                ON c.id = cci.contest_id)
        LEFT JOIN 
            cvr_contest_info_choice as ccic 
            ON ccic.cvr_contest_info_id = cci.id)
            ON ccd.contest_id = c.id 
        AND ccd.choice = ccic.choice
    GROUP BY contest, candidate
    ORDER BY contest, candidate
;
 contest |   candidate    | votes 
---------+----------------+-------
      71 | Clear Winner   |   123
      71 | Distant Loser  |    14
      72 | Janet Lee Cook |    27
      72 | Jeff Baker     |    12
      73 | NO             |    31
      73 | YES            |    30
(6 rows)

